### PR TITLE
Adds sideboard.lib.profile decorator to renderable uber endpoints

### PR DESCRIPTION
**Pull request by RobRuana**

### CODE REVIEW

File: uber/configspec.ini [hunk 1]
```diff
@@ -7,6 +7,10 @@
 # after a deadline or on-site.
 dev_box = boolean(default=True)
 
+# Redirect 404s to Uber's default URL
+default_url = string(default="/uber/common/")
+default_url_priority = integer(default=1)
```
Comment (RobRuana) [inline uber/configspec.ini]: Our nginx config will now pass requests for "/profiler" through to uber. We're setting the `default_url` so those requests will just forward to "/uber/common".

Comment (kitsuta) [inline uber/configspec.ini]: Hm, don't we already have this in the form of `path` and `url_root` and `url_base`?

Comment (RobRuana) [inline uber/configspec.ini]: Ah, good catch. `path` and `default_url` do different things, so we still need `default_url` (it's a actually a sideboard setting that plugins can override). But `default_url` should really use the interpolated value of `path`.

**Comment by EliAndrewC** [comment]
+1 (belatedly)

**Pull request merged**